source/guides/core-concepts/retry-ability.md

```diff
@@ -5,13 +5,18 @@ title: Retry-ability
 {% note info %}
 # {% fa fa-graduation-cap %} What you'll learn
 
+- How Cypress retries tests
 - How Cypress retries commands and assertions
 - When commands are retried and when they are not
 - How to address some situations of flaky tests
 {% endnote %}
 
 A core feature of Cypress that assists with testing dynamic web applications is retry-ability. Like a good transmission in a car, it usually works without you noticing it. But understanding how it works will help you write faster tests with fewer run-time surprises.
 
+# Test retries
+
+Tests retry by default during `cypress run` if they fail. Additionally, the number of retries can be configured. Read {% url "Test Retries" test-retries %} for more information.
+
```

Comment:
I think we need more of a delineation between what Test Retries are versus Retries _in_ tests. I struggled with the context switch. I think we at least need a structural change to delineate entire tests being retried versus the commands within the test being retried. @bencodezen

Reply:
Yah, I added this small section as an afterthought when looking through the rest of our docs - thinking where test retries should be mentioned. Definitely 'Retry-ability' doc should be updated to incorporate this concept more clearly.

Reply:
New changes made!